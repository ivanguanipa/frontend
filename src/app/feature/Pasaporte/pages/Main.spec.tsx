import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import MainPage  from './Main';
import { Route, Switch } from 'react-router-dom';

describe('Main Test', () => {
  let componentWrapper: ShallowWrapper;

  afterEach(() => {
    componentWrapper.unmount();
  });

  it('should match snapshot', () => {
    componentWrapper = shallow(
        <Route path="/pasaportes" exact component={MainPage}></Route>
    );
    expect(componentWrapper).toMatchSnapshot();
  });
});

export {};
