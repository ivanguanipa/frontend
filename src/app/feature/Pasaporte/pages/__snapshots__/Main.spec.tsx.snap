// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Test should match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
